integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -306,13 +306,9 @@ public int create(String path, @mode_t long mode, FuseFileInfo fi) {
           CreateFilePOptions.newBuilder()
               .setMode(new alluxio.security.authorization.Mode((short) mode).toProto())
               .build());
-      synchronized (mOpenFiles) {
-        mOpenFiles.add(new OpenFileEntry(mNextOpenFileId, path, null, os));
-        fi.fh.set(mNextOpenFileId);
-
-        // Assuming I will never wrap around (2^64 open files are quite a lot anyway)
-        mNextOpenFileId += 1;
-      }
+      long fid = mNextOpenFileId.getAndAdd(1);
```

Comment:
is there a reason to not use `getAndIncrement`?

Reply:
changed